example/lib/main.dart

```diff
@@ -56,26 +56,49 @@ class _MyHomePageState extends State<MyHomePage> {
       appBar: AppBar(
         title: Text(widget.title),
       ),
-      body: BarcodeScannerWidget(
-        controller: _controller,
-        onScan: (code) {
-          print("GOT BARCODE =========== ${code.code}");
-        },
-        configuration: const ScannerConfiguration(
-          enableFormats: {BarcodeFormat.qr},
-          cameraConfiguration: CameraConfiguration(
-            frameRate: 30,
-            mode: BarcodeDetectionMode.continuous,
-            resolution: CameraResolution.medium,
-            type: CameraType.back,
+      body: Column(
+        children: [
+          Expanded(
+            child: BarcodeScannerWidget(
+              controller: _controller,
+              onScan: (code) {
+                print("GOT BARCODE =========== ${code.code}");
+              },
+              configuration: const ScannerConfiguration(
+                enableFormats: {BarcodeFormat.qr},
+                cameraConfiguration: CameraConfiguration(
+                  frameRate: 30,
+                  mode: BarcodeDetectionMode.continuous,
+                  resolution: CameraResolution.medium,
+                  type: CameraType.back,
+                ),
+              ),
+              scanners: [
+                FastBarcodeScanner(),
+                HoneywellBarcodeScanner(),
+                UnitechBarcodeScanner(),
+                BlueBirdBarcodeScanner(),
+                ZebraBarcodeScanner('my_profile'),
+              ],
+            ),
+          ),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+            children: [
+              MaterialButton(
```

Comment:
if (controller.hasTorch)

Reply:
There is no hasTorch or supportsTorch atm

Reply:
Maybe we should have that as well, no?

Reply:
We can add TorchCompat, but the fast_barcode_scanner does not provide a function for this atm